packages/core/src/app/ui/dropdown/DropdownTrigger.tsx

```diff
@@ -96,7 +96,7 @@ export default class DropdownTrigger extends Component<DropdownTriggerProps, Dro
         }
 
         this.setState({ shouldShow: false }, () => {
-            document.removeEventListener('click', this.handleClose);
+            document.getElementById('checkout-app')?.removeEventListener('click', this.handleClose);
```

Comment:
Since version 17, React has changed event delegation, and `document.addEventListener()` no longer works.
https://legacy.reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation